cmd/clef/main.go

```diff
@@ -351,6 +378,61 @@ func attestFile(ctx *cli.Context) error {
 	return nil
 }
 
+func listAccounts(c *cli.Context) error {
+	if err := initialize(c); err != nil {
+		return err
+	}
+	// listaccounts is meant for users using the CLI.
+	var (
+		ui                        = core.NewCommandlineUI()
+		pwStorage storage.Storage = &storage.NoStorage{}
+		ksLoc                     = c.String(keystoreFlag.Name)
+		lightKdf                  = c.Bool(utils.LightKDFFlag.Name)
+	)
+	am := core.StartClefAccountManager(ksLoc, true, lightKdf, "")
+	// Access external API and call List()
+	api := core.NewSignerAPI(am, 0, true, ui, nil, false, pwStorage)
+	internalApi := core.NewUIServerAPI(api)
+	accs, err := internalApi.ListAccounts(context.Background())
+	if err != nil {
+		return err
+	}
+	if len(accs) == 0 {
+		fmt.Println("\nThe keystore is empty.")
+	}
+	fmt.Println()
+	for _, account := range accs {
+		fmt.Println(account.Address)
+	}
+	return err
+}
+
+func listWallets(c *cli.Context) error{
+	if err := initialize(c); err != nil {
+		return err
+	}
+	// listaccounts is meant for users using the CLI.
+	var (
+		ui                        = core.NewCommandlineUI()
+		pwStorage storage.Storage = &storage.NoStorage{}
+		ksLoc                     = c.String(keystoreFlag.Name)
+		lightKdf                  = c.Bool(utils.LightKDFFlag.Name)
+	)
+	am := core.StartClefAccountManager(ksLoc, true, lightKdf, "")
+	// Access external API and call List()
+	api := core.NewSignerAPI(am, 0, true, ui, nil, false, pwStorage)
+	internalApi := core.NewUIServerAPI(api)
+	wallets := internalApi.ListWallets()
+	if len(wallets) == 0 {
+		fmt.Println("\nThere are no wallets.")
+	}
+	fmt.Println()
+	for _, wallet := range wallets {
+		fmt.Println(wallet)
+	}
```

Comment:
Here too, we have a lot more info. Let's provide it2
```suggestion
	for i, wallet := range wallets {
		fmt.Printf("%d. Keystore at %v (%v %v)\n", i, wallet.URL, wallet.Status, wallet.Failure)
		for _, acc := range wallet.Accounts {
			fmt.Printf("\t%v (%v)\n", acc.Address, acc.URL)
		}
	}
```